sema/ro/creator/roc.py

```
@@ -0,0 +1,73 @@
+            with open(rocrate_ignore_file, "r") as f:
+                return [line.strip() for line in f if line.strip() and not line.startswith("#")]
+        self.rocrate_ignore.extend(self._parse_rocrate_ignore())
+        return []
+
+    def _generate_default_metadata(self):
+        for p in self.rocrate_path.glob("**/*"):
+            p = p.relative_to(self.rocrate_path).as_posix()
+            if not any(re.search(pattern, p) for pattern in self.rocrate_ignore):
+                self.metadata[p] = None
+
+    def _generate_blueprint_metadata(self):
```

Ruff (0.12.2)

Missing return type annotation for private function `_generate_blueprint_metadata`

Add return type annotation: `None`

(ANN202)